src/transformStateWithClones.js

```diff
@@ -6,8 +6,59 @@
  *
  * @return {Object[]}
  */
+function transform(state, action) {
+  const cloneState = {};
+
+  for (const key in state) {
+    cloneState[key] = state[key];
+  }
```

Comment:
```suggestion
  const cloneState = { ...state };
```
use spread operator to make copy